test/verifiedTests/analysis/testFASTCORE/LP9test/testLP9.m

```diff
@@ -0,0 +1,51 @@
+% The COBRAToolbox: Lp9.m
+%
+% Purpose:
+%     - test if bug fix in LP9 affects the results for test model (without
+%     coupling constraints)
+%
+% Authors:
+%     - fix compatibility for models with coupling constraints: Agnieszka
+%     Wegrzyn 2016/03/06
+%
+
+global CBTDIR
+
+% define the features required to run the test
+requiredSolvers = {'gurobi'};
+
+% require the specified toolboxes and solvers, along with a UNIX OS
+solversPkgs = prepareTest('requiredSolvers', requiredSolvers);
+
+% save the current path and initialize the test
+currentDir = cd(fileparts(which(mfilename)));
+
+% determine the test path for references
+testPath = pwd;
+
+% load the model
+model = getDistributedModel('ecoli_core_model.mat'); %For all models in the test/models folder and subfolders
+
+% load input variables for the function
+load('options.mat');
+
+% load reference data
+load('reference_V.mat');
+
+% load 
+for k = 1:length(solversPkgs.LP)
+    fprintf(' -- Running testLP9.m using the solver interface: %s ... ', solversPkgs.LP{k});
+
+    solverLPOK = changeCobraSolver(solversPkgs.LP{k}, 'LP', 0);
+
+    if solverLPOK
+        % created test data for new version
+        V_test = LP9(options.K, options.P, model, options.LPproblem, options.epsilon);
+    end
+    assert(isequal(reference_V,V_test), '\nResults (V) are not consistent between old and new version of LP9\n')
```

Comment:
Instead of using here `isequal`, you can set a `tolerance` such as explained [here](https://opencobra.github.io/cobratoolbox/stable/contributing.html#test-if-an-output-is-correct).